index.html

```diff
@@ -1,96 +1,130 @@
-                       </em>
-                        <!-- lettel line  -->
-                             <div class="line"></div>
-                        <!-- end of lettel line  -->
-                        <p>A computer would deserve to be called intelligent if it could deceive a human into believing that it was human</p>
-                   </div>
-               </div>
-               <!-- end feature cars here -->
-           </div>
-            <!-- show more btn -->
-            <button type="button" onclick="showMore()">MORE <img src="Resources/images/icons8-expand-arrow-50.png" alt=""></button>
-            <!-- end of show more btn-->
-        </section>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="home.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <div class="topMenu">
+        <nav id="topNav">
+          <ul>
+            <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href=""><i class="fab fa-twitter"></i></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="mainMenu">
+        <div class="mainMenuCont">
+          <div class="logo">
+            <a href="index.html"><img src="Resources/images/my-logo.png" alt=""></a>
+          </div>
+          <i id="menuBtn" class="fas fa-bars fa-lg"></i>
+          <nav id="mainNav">
+            <i id="closeBtn" class="fas fa-times fa-lg"></i>
+            <ul>
+              <li><a class="aboutLink" href="about.html">About</a></li>
+              <li><a href="#mainProgram">Program</a></li>
+              <li><a href="#featuredSpeakers">Speakers</a></li>
+              <li><a href="#partners">Partners</a></li>
+              <li><button type="button" class="mainNavBtn">FASTDEVZ</button></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <section class="hero">
+      <div class="heroCont">
+        <div class="mainHeading">
+          <h2>"Welcome To The World of Infinity Creations"</h2>
+          <h1><br>SOFTWARE IS A GREAT COMBINATION BETWEEN ARTISTRY AND ENGINEERING.<br>2022</h1>
+        </div>
+        <div class="message">
+          <p>
+            Excellent customer service is the number one job in any company! It is the personality of the company and the reason customers come back. Without customers there is no company. (the customer is king at fastdevz)
+          </p>
+        </div>
```

Comment:
- [ ] [Optional] There is no need wrapping a `<p>` tag in a `<div>` container as the`<p>` tag is also a block-level element.

Reply:
@phelian23 is also done here I removed the p tag